andela/travel_tool_front
# #162303455 Outflow for travel-readiness should have departure dates instead

#### What does this PR do?
In the travel-readiness card, outflows should have ```Expected Departure Date```, not ```Expected Arrival Date``` as it is for inflows.

#### Description of Task to be completed?
- Update the TravelReadiness component to render a conditional value depending on the ```travelFlow``` state

#### How should this be manually tested?
* Clone the repos: ``` https://github.com/andela/travel_tool_front ``` and ``` https://github.com/andela/travel_tool_back ```.
* cd into their respective directories ```travel_tool_front``` and ```travel_tool_back```.
* Checkout to the branches ```ch-modify-readiness-outflow-data-162303455 ``` on ``` travel_tool_front ``` and ```ch-modify-readiness-outflow-data-162303455``` on ``` travel_tool_back ```.
* Start up the back-end and front-end with ``` yarn start:dev ``` and ``` yarn start ``` respectively.
* Login to the app on the front end,
* Make requests. Make sure to use your location as the origin. (As a travel Admin, this will enable you see them as outflows when they have been approved).
* Approve these requests.
* Navigate to the ``` Dashboard ``` option and select ‘outflow’ on the ```Travel Readiness``` card
* You should see ```Expected Departure Date``` instead of ```Expected Arrival Date```.
* You should also see the same title when you export the ```outflow``` file

#### Any background context you want to provide?
* You should be logged in as a Travel Admin to view the dashboard.

#### What are the relevant pivotal tracker stories?
[162303455](https://www.pivotaltracker.com/story/show/162303455)

#### Screenshots (if appropriate)
<img width="738" alt="screen shot 2018-11-29 at 1 57 03 pm" src="https://user-images.githubusercontent.com/29521319/49223316-ba60de80-f3de-11e8-802a-be657c5d5004.png">


#### Questions:
